components/ai_chat/core/DEPS

```diff
@@ -0,0 +1,4 @@
+include_rules = [
+  "+services/network/public",
+  "+absl/types/optional.h",
```

Comment:
I haven't looked but are both of these still needed in `components/ai_chat/browser/DEPS`?